esterni/piano-di-progetto/components/analisi-dei-rischi.tex

```diff
@@ -105,6 +105,12 @@ \subsection{Elenco dei rischi}%
   Monitoraggio:&
   \multicolumn{3}{p{38.5em}|}{Il proponente notificherà il gruppo in caso di problemi se possibile e in ogni caso la copia dell'applicazione sul server non sarà l'unica copia disponibile.}\\
   \hline % chktex 44
+  Problemi di comunicazione fra i membri di GruppOne \textbf{RPER004}& Essendo questo il primo progetto fortemente incentrato nella collaborazione per molti dei componenti del gruppo, si potranno presentare situazioni di incomprensione, dovute ad una poca od inefficace comunicazione fra i membri. &
+  Il gruppo predisporrà diversi canali di comunicazione, per invitare i membri del gruppo a tenersi aggiornati sull'andamento del progetto.&
+  \textbf{Probabilità: }  Media \textbf{Impatto: } Significativo\\
+  Monitoraggio:&
+  \multicolumn{3}{p{38.5em}|}{Periodicamente, il gruppo svolgerà incontri (possibilmente di persona, altrimenti in modalità telematica) atti a valutare l'andamento del progetto.}\\
+  \hline % chktex 44
 \end{longtable}
```

Comment:
Propongo, se possibile e se ha senso, di aggiungere un altro rischio, legato alla pandemia COVID-19, che ci ha costretto a non vederci fisicamente e lavorare in modalità remota. (Spingo su questo perchè all'ultimo colloquio tenuto con Vardanega era stato chiesto esplicitamente ad ognuno di noi se questo avesse potuto creare particolari problemi).
Aggiungo in seguito quello che potrebbe essere inserito nella tabella:

**Rischio:** 
Problemi Sanitari
**Descrizione:** 
Il gruppo potrebbe non vedersi fisicamente per lavorare insieme, a causa del diffondersi del virus COVID-19 e della proroga del decreto che vieta spostamenti ai cittadini nel suolo italiano.
**Pianificazione:** 
Il team ha affermato che l'impossibilità di movimento non è un impedimento tale che possa compromettere la continuazione del progetto didattico.
**Indicatori:** 
_Probabilità:_ Alta  
_Impatto:_ Tollerabile

Reply:
Siccome non avremmo mai potuto prevedere come rischio "pandemia a livello globale", non ha molto senso metterlo come rischio previsto nell'analisi. Siccome l'unico effetto della pandemia per noi è che non possiamo vederci di persona, nell'attualizzazione l'ho segnato come evenienza di RPER004 (problemi di comunicazione).

Reply:
considero risolto